head.html

```diff
@@ -2,3 +2,5 @@
 <script src="/scripts/lib-franklin.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
 <link rel="stylesheet" href="/styles/styles.css"/>
+<script type="text/javascript" async="async" src="//assets.adobedtm.com/launch-ENd6668700e6ad4d64a37c3d34c489ee03.min.js?ver=6.2.2" id="analytics_tracking_script-js"></script>
+<script src="/scripts/analytics.js" type="module"></script>
```

Comment:
We should be loading the javascript in the `delayed.js` script, this would ensure there is no impact on the lighthouse score.

Documentation : https://www.hlx.live/developer/keeping-it-100#phase-d-delayed